web/src/styles/_sass-files/variables.scss

```diff
@@ -5,6 +5,13 @@ $sm: 40em; //tablet + up (640px)
 $md: 50em; //big tablet (800px)
```

Comment:
[dust] curious why the underscore for this folder name

Reply:
I'm doing to remove it.